MaterialDesignThemes.Wpf/DrawerHost.cs

```diff
@@ -57,6 +56,9 @@ public class DrawerHost : ContentControl
         private FrameworkElement _rightDrawerElement;
         private FrameworkElement _bottomDrawerElement;
 
+        private DrawerOpenedEventHandler _attachedDrawerOpenedEventHandler;
+        private DrawerClosingEventHandler _attachedDrawerClosingEventHandler;
```

Comment:
I don't think we need the attached event handlers for the drawer host. These were a bit of a hack for the dialog host, and tend to cause as many problems as they solve.